### PR TITLE
Volumetric support for MeasureObjectSizeShape

### LINKED ISSUE #2459: MeasureObjectSizeShape should support three-dimensional images

### PR DESCRIPTION
**Pull request by mcquin**
* area
* extent
* perimeter
* center_z

Resolves #2459

### CODE REVIEW

File: cellprofiler/modules/measureobjectsizeshape.py [hunk 1]
```diff
@@ -285,120 +287,192 @@ def run(self, workspace):
...
-                      (F_MAX_FERET_DIAMETER, max_feret_diameter)] +
-                         [(self.get_zernike_name((n, m)), zf[(n, m)])
-                          for n, m in zernike_numbers]):
-            self.record_measurement(workspace, object_name, f, m)
+            labels = objects.segmented
+
+            props = skimage.measure.regionprops(labels)
+
+            # Area
+            areas = [prop.area for prop in props]
```
Comment (0x00b1) [inline cellprofiler/modules/measureobjectsizeshape.py]: We should consolidate similar accumulations (e.g. `areas`, `extents`, and `centers`) into one iterator.

File: cellprofiler/modules/measureobjectsizeshape.py [hunk 2]
```diff
@@ -285,120 +287,192 @@ def run(self, workspace):
...
+
+            self.record_measurement(workspace, object_name, F_AREA, areas)
+
+            # Bounding box (min_z, min_x, min_y, max_z, max_x, max_y)
```
Comment (0x00b1) [inline cellprofiler/modules/measureobjectsizeshape.py]: This is unused.

Comment (0x00b1) [inline cellprofiler/modules/measureobjectsizeshape.py]: It’d be nice to have a few additional properties on `cellprofiler.object.Objects` for common computations.

File: cellprofiler/modules/measureobjectsizeshape.py (hunk 1 above)
Comment (mcquin) [inline cellprofiler/modules/measureobjectsizeshape.py]: #2450

**Pull request merged**